code/game/dna/genes/vg_powers.dm

```diff
@@ -121,6 +121,20 @@ Obviously, requires DNA2.
 		return 0
 	return ..(M,flags)
 
+// NOIR
+
+/obj/screen/plane_master/noir_master
+	plane = PLANE_NOIR_BLOOD
+	color = list(1,0,0,0,
+				 0,1,0,0,
+				 0,0,1,0,
+				 0,0,0,0,
+				 0,0,0,1)
+	appearance_flags = NO_CLIENT_COLOR
```

Comment:
You should read your code again and be ashamed.

*also that piss poor copypaste smh*